helmfile.d/upstream/opensearch-project/opensearch-dashboards/templates/serviceMonitor.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.serviceMonitor.enabled }}
```

Comment:
I see that they've added servicemonitors for opensearch+dashboards.
Do we have a list of metrics that they expose to see if this can obsolete the prometheus-opensearch-exporter?

Reply:
Yeah I noticed this when I started with this upgrade but got side-tracked by all the other things, so I never got the chance to look into it. Our focus in the GoTo area right now is to replace our own charts with built-in functionality and simplify our setup overall, so this is definitely something we would want to investigate.